src/Client/StubStripeClient.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace MatchBot\Client;
+
+use phpDocumentor\Reflection\Types\Never_;
+use Stripe\PaymentIntent;
+
+/**
+ * Does not connect to stripe. For use in load testing to allow testing Matchbot with high traffic levels
+ * without sending all that traffic to Stripe. Test mode stripe does not allow sending high volumes of test traffic
+ * so we have to stub it out.
+ */
+class StubStripeClient implements Stripe
```

Comment:
and this one perhaps also doesn't need tests of its own as its only for use in testing. Although it might be worth having unit tests to verify that we're don't have a bug that would get in the way of running load tests. If we do need to unit test it then we'll have to get rid of the direct `usleep` call, probably by using the Symfony Clock component again.